src/en/proposals/fairlysadpanda-revolutionaries.md

```diff
@@ -0,0 +1,275 @@
+
+## Goals
+
+### Spread The Revolution
+
+Use Propaganda posters and literature to spread the Influence of the Revolution and get into the harder-to-reach areas of the ship.
+
+### Free Yourself From Tyranny
+
+Ambush and arrest your department head; steal their ID for the greater good, and requisition their tools for the Revolution.
+
+### Don’t Get Gibbed By Sec
+
+Avoid being rumbled by Sec; they have guns and equipment that makes rumbling and smashing your mutiny possible, and it’s hard (although not impossible) to flip Sec members to your side.
+
+Core Mechanics
+
+### Propaganda
+
+The printers, posters and books of the station can be influenced by Revolutionaries, who can also make them via crafting and using the station’s printers. Sec, the HoP and the Captain are trained to identify revolutionary literature (it’s not obvious to the rest of the station), but it’s also possible for traitors and nukies to make them to throw Sec off their own activities.
+
+Propaganda slowly ticks up an exposed character’s Influence meter and allows Head Revs to quickly convert a compromised department if not dealt with.
+
+### Drugs
+
+Mindbreaker Toxin can be manufactured and administered as a bulk Influence dose.
+
+### Brainwashing
+
+Converted Revolutionaries have an obvious trait that can be inspected:
+
+“They appear slightly dazed and confused”.
+
+This state is shared with imbiding some contraband chemicals, like Mindbreaker Toxin: being dazed and confused itself is not evidence of the round type being Revolutionaries.
+
+It’s possible to hide this dazed state by covering the character’s eyes via gas masks, welding masks, sunglasses, etc.
+
+# Role Outline: The Security Team
+
+## How It Should Feel
+
+The crew might be plotting against you. Is the missing Chief Engineer due to incompetence, a traitor, or a mutiny? Dispatch officers to identify suspicious individuals - people who are in places they shouldn’t be, and confused people acting out-of-character. Find the spiders at the centre of the web and crush them. Or, robust your way to victory by reclaiming the Bridge from the mutineers.
+
+## Goals
+
+### Investigate Insurrection
+
+Why are your posters being torn down? Where’s the HoP’s ID? Why does that Clown seem to show up everywhere?
+
+### Crush The Mutiny
+
+Once you’re confident it’s a Rev round - the HoS being missing or his ID being stolen is a bit of a clue - you need to find out who it is who’s been doing all the converting and stop them. That’s easier said than done - even de-converted revolutionaries might not know who the ringleaders are, but arresting and interviewing them after deconversion is a good place to start.
+
+### Reclaim The Bridge
+
+No matter what, if things go wrong there’s always violence to fall back on. Grab your hardsuits and laser rifles and reclaim the bridge from those traitorous dogs.
+
+## Core Mechanics
+
+### Propaganda
```

Comment:
Really fun idea, but again, like above, will definitely be "metagamed." If you are walking around and see posters to join a revolution, you are going to call a revolution. I daresay at this point that this is not metagaming because it has a completely justified IC reason. I cannot see any reason why these posters would be used outside of a revolution, making them completely obvious when there is one.

Reply:
Misdirection by other antags and "funny comedy" by clowns and mimes aside, this is a fair point.

Reply:
Anything used as rev branding should be something that (at least at the start) can be masked as normal non-rev branding. That way it cannot be instantly meta'd as Revs. Could add plenty of options for service staff to add other propaganda such as TV presentations.

Reply:
Still think we need more propaganda than posters, other methods that arnt as overt. Could be as suggested on the discord as some sort of code phrase / chant the HRs can call out to cause influence gain, for example.